core/examples/json/defaultSortOrder.json

```diff
@@ -0,0 +1,14 @@
+{
+    "id": "sentinel1cat",
+    "itemType": "record",
+    "title": "Sentinel-1 Catalogue",
+    "description": "A sample catalogue of Sentinel-1 products stored in S3 on AWS.",
+    "defaultSortOrder": [ "updated-", "area-"],
```

Comment:
The sortby parameter prepends the order indicator. To be consistent this should be `[ "-updated", "-area"]`.

Reply:
Doh!  You are right ... I fix!